### DISCUSSION TITLE
Try out the closed issue #409

### DISCUSSION THREAD

**Discussion by Djokix98** [atteneder/glTFast#442] [General]
Hello, as far as i can see the issue #409 was marked as Closed and set as a part of the 5.0.0 Milestone. However the latest release is 4.8.3 which doesnt have this change yet so my question would be, is there a way to try out #409 somehow before the 5.0.0 Milestone is reached?

Thanks in advance,
Djordje

**Comment by atteneder** [comment]
You can use/test a certain (development) version of glTFast by:

- Cloning the repository (the main branch or a certain commit/version)
- Install the package from the local path you just cloned it into
- Run your project

5.x is delayed since I've been offline for some weeks. Thanks for your patience.
